cmake/kokkos_tribits.cmake

```diff
@@ -448,6 +448,13 @@ FUNCTION(KOKKOS_INTERNAL_ADD_LIBRARY LIBRARY_NAME)
     ${PARSE_SOURCES}
   )
 
+  IF(PARSE_SHARED OR BUILD_SHARED_LIBS)
+    SET_TARGET_PROPERTIES(${LIBRARY_NAME} PROPERTIES
+      VERSION   ${Kokkos_VERSION}
+      SOVERSION ${Kokkos_VERSION_MAJOR}.${Kokkos_VERSION_MINOR}
+    )
+  ENDIF()
```

Comment:
https://github.com/kokkos/kokkos/blob/78538c8574777342f64c7da657f419778eaa2a10/core/src/CMakeLists.txt#L75
becomes redundant with the changes suggested.  You might want to remove